### PR TITLE
Add latest Ulster Scots translations for the Census HH/HI schemas

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?

Updates the individual & household .po file with the latest translations in Ulster Scots (eo) from Crowdin.

### How to review

Check the translations match Crowdin.

### CODE REVIEW

**Comment by petechd** [comment]
[Load Ulster Scots PO files for NISRA](https://trello.com/c/hREqXUon/4416-load-ulster-scots-po-files-for-nisra)

File: translations/eo/census_household_gb_nir_eo.po [hunk 1]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
+msgstr "In thair foremaist tack, whut wus <em>{person_name_possessive}</em> stannin in wark?"
+
+#. Question text
 msgid "What is the name of the organisation or business you work for?"
-msgstr "Whut dae the' cry tha curn ur consarn ye wark fur?"
+msgstr "Whut dae the' cry tha consarn or ootfit ye wark til?"
+
+#. Question text
+msgid "What is the name of the organisation or business <em>{person_name}</em> works for?"
+msgstr "Whut dae the' cry tha consarn or ootfit<em>{person_name}</em> warks til?"
```
Comment (berroar) [inline translations/eo/census_household_gb_nir_eo.po:1353]:
```suggestion
msgstr "Whut dae the' cry tha consarn or ootfit <em>{person_name}</em> warks til?"
```

File: translations/eo/census_household_gb_nir_eo.po [hunk 2]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
-#. answer-id: disability-other-answer
-msgctxt "Answer for: Does {person_name} have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
+#. Answer option
+msgctxt "Does {person_name} have any of the following <em>other health conditions</em> which have lasted, or are expected to last, at least 12 months?"
 msgid "An emotional, psychological or mental health condition"
-msgstr "A trouble 'at's hairt-scaudin, ur wammlin in tha mine ur harns"
+msgstr "An emotional psychological or mental halth condeetion"
 
-#. answer-id: disability-other-answer
-msgctxt "Answer for: Does {person_name} have any of the following <em>other heal
```
Comment (berroar) [inline translations/eo/census_household_gb_nir_eo.po:6210]:
```suggestion
msgstr "{second_person_name} bes yer <em>step-brither or step-sister</em>"
```

File: translations/eo/census_household_gb_nir_eo.po [hunk 3]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
+msgid "One year ago, what was <em>{person_name_possessive}</em> usual address?"
+msgstr "A yeir syne, whut wus tha address o <em>{person_name_possessive}</em> fur ordnar?"
+
+#. Question text
+msgid "In which country outside the UK was your usual address one year ago?"
+msgstr "Whut kintrie ootbye tha UK wus yer bidin place a yeir syne?"
 
+#. Question text
+msgid "In which country outside the UK was <em>{person_name_possessive}</em> usual address one year ago?"
+msgstr "In whutch kintre ootbye tha UK wus <em>{person_name_possessive}</em>address fur ordnar yin yeir syne?"
```
Comment (berroar) [inline translations/eo/census_household_gb_nir_eo.po:1025]:
```suggestion
msgstr "In whutch kintre ootbye tha UK wus <em>{person_name_possessive}</em> address fur ordnar yin yeir syne?"
```

File: translations/eo/census_household_gb_nir_eo.po [hunk 4]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
+msgid "Briefly describe what <em>{person_name}</em> does in their main job."
+msgstr "Gie shoart wittins anent whut <em>{person_name}</em> daes in thair foremaist tack."
+
+#. Question text
 msgid "Briefly describe what you did in your main job."
 msgstr "Gie shoart wittins anent whut ye daed in yer foremaist tack."
 
+#. Question text
+msgid "Briefly describe what <em>{person_name}</em> did in their main job."
+msgstr "Gie shoart wittins anent whut <em>{person_name}</em>daed in thair foremaist tack."
```
Comment (berroar) [inline translations/eo/census_household_gb_nir_eo.po:1393]:
```suggestion
msgstr "Gie shoart wittins anent whut <em>{person_name}</em> daed in thair foremaist tack."
```

File: translations/eo/census_household_gb_nir_eo.po [hunk 5]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
-msgstr "Gin it's set, tha kine o tha lanlaird, tha like o a hoosin join, cooncil ur hiddlie"
+#. Question text
+msgid "What is the address of your main place of <em>work</em>?"
+msgstr "Whut bes tha address o yer foremaist place o <em>wark</em>?"
 
-msgid "Type of central heating"
-msgstr "Whut soart o yin-soorce haitin"
+#. Question text
+msgid "What is the address of <em>{person_name_possessive}</em> main place of <em>work</em>?"
+msgstr "Whut bes tha address o <em>{person_name_possessive}</em>foremaist place o <em>wark</em>?"
```
Comment (berroar) [inline translations/eo/census_household_gb_nir_eo.po:1513]:
```suggestion
msgstr "Whut bes tha address o <em>{person_name_possessive}</em> foremaist place o <em>wark</em>?"
```

File: translations/eo/census_household_gb_nir_eo.po [hunk 6]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
-msgstr "Rinnins anent tha bodie's sel, tha like o tha date boarn, in whut kintrie, an releegion"
+#. Question text
+msgid "What was the address of your main place of <em>work</em>?"
+msgstr "Whut wus tha address o tha foremaist place ye <em>wrocht</em>?"
 
-msgid "Main language"
-msgstr "Foremaist leid"
+#. Question text
+msgid "What was the address of <em>{person_name_possessive}</em> main place of <em>work</em>?"
+msgstr "Whut wus tha address o <em>{person_name_possessive}</em>foremaist place o <em>wark</em>?"
```
Comment (berroar) [inline translations/eo/census_household_gb_nir_eo.po:1521]:
```suggestion
msgstr "Whut wus tha address o <em>{person_name_possessive}</em> foremaist place o <em>wark</em>?"
```

File: translations/eo/census_household_gb_nir_eo.po [hunk 7]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
-msgstr "Cheynge rinnins fur {person_name}"
+#. Question text
+msgid "What is the address of your main place of <em>study</em>?"
+msgstr "Whut wud be tha address o tha foremaist place ye hae yer <em>lear</em>?"
 
-msgid "Does anyone usually live at {address}?"
-msgstr "Daes oniebodie bide fur ordnar at {address}?"
+#. Question text
+msgid "What is the address of <em>{person_name_possessive}</em> main place of <em>study</em>?"
+msgstr "Whut bes tha address o <em>{person_name_possessive}</em>foremaist place o <em>lear</em>?"
```
Comment (berroar) [inline translations/eo/census_household_gb_nir_eo.po:1569]:
```suggestion
msgstr "Whut bes tha address o <em>{person_name_possessive}</em> foremaist place o <em>lear</em>?"
```

File: translations/eo/census_household_gb_nir_eo.po [hunk 8]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
+msgctxt "Where is your main place of <em>work</em>?"
+msgid "Answer for the place where you spend the most time. Even if ill, on maternity leave, holiday or temporarily laid off provide details of your main place of work."
+msgstr "Gie repone fur tha place whar ye ware tha maist. Even gif yer seek, aff wark fur rairin a new babbie, oan hoaliday or by whyles laid aff gie rinnins anent yer foremaist place o wark."
 
-msgid "{person_name} is {age_in_years} years old. Is this correct?"
-msgstr "{person_name} bes {age_in_years} yeir auld. Wud this be richt?"
+#. Question description
+msgctxt "Where is <em>{person_name_possessive}</em> main place of <em>work</em>?"
+msgid "Answer for the place where <em>{person_name}</em> spends the most time. Even if ill, on maternity leave, holiday or temporarily laid off provide details of their main place of work."
+msgstr "Gie repone fur tha place whar <em>{person_name}</em>wares tha maist. Even gif thair seek, aff wark fur rairin a new babbie, oan hoaliday or by whyles laid aff gie rinnins anent thair foremaist place o wark."
```
Comment (berroar) [inline translations/eo/census_household_gb_nir_eo.po:1764]:
```suggestion
msgstr "Gie repone fur tha place whar <em>{person_name}</em> wares tha maist. Even gif thair seek, aff wark fur rairin a new babbie, oan hoaliday or by whyles laid aff gie rinnins anent thair foremaist place o wark."
```

File: translations/eo/census_household_gb_nir_eo.po [hunk 9]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
+
+#. Question description
+msgctxt "Did <em>{person_name}</em> supervise or oversee the work of other employees on a day-to-day basis?"
+msgid "This could be remotely or in person"
+msgstr "This cud be frae far aff or oan tha steidin"
+
+#. Question description
+msgctxt "In your main job, how many hours a week do you usually work?"
+msgid "If the <strong>coronavirus</strong> pandemic has affected your working hours, select the answer that best describes your <strong>current circumstances</strong>"
+msgstr "Gin tha <strong>coronavirus</strong>smittal haes made odds til yer oors o wark, ootwale tha repone 'at gies a richt insensin o yer<strong> stannin thenoo</strong>"
```
Comment (berroar) [inline translations/eo/census_household_gb_nir_eo.po:1749]:
```suggestion
msgstr "Gin tha <strong>coronavirus</strong> smittal haes made odds til yer oors o wark, ootwale tha repone 'at gies a richt insensin o yer<strong> stannin thenoo</strong>"
```

File: translations/eo/census_individual_gb_nir_eo.po [hunk 10]
```diff
@@ -2,3312 +2,4784 @@ msgid ""
...
+
+#. Question description
+msgctxt "Did <em>{person_name}</em> supervise or oversee the work of other employees on a day-to-day basis?"
+msgid "This could be remotely or in person"
+msgstr "This cud be frae far aff or oan tha steidin"
+
+#. Question description
+msgctxt "In your main job, how many hours a week do you usually work?"
+msgid "If the <strong>coronavirus</strong> pandemic has affected your working hours, select the answer that best describes your <strong>current circumstances</strong>"
+msgstr "Gin tha <strong>coronavirus</strong>smittal haes made odds til yer oors o wark, ootwale tha repone 'at gies a richt insensin o yer<strong> stannin thenoo</strong>"
```
Comment (berroar) [inline translations/eo/census_individual_gb_nir_eo.po:1279]:
```suggestion
msgstr "Gin tha <strong>coronavirus</strong>smittal haes made odds til yer oors o wark, ootwale tha repone 'at gies a richt insensin o yer<strong> stannin thenoo</strong>"
```
```suggestion
msgstr "Gin tha <strong>coronavirus</strong> smittal haes made odds til yer oors o wark, ootwale tha repone 'at gies a richt insensin o yer<strong> stannin thenoo</strong>"
```

File: translations/eo/census_household_gb_nir_eo.po [hunk 11]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
+
+#. Question description
+msgctxt "In <em>{person_name_possessive}</em> main job, how many hours a week do they usually work?"
+msgid "If the <strong>coronavirus</strong> pandemic has affected their working hours, select the answer that best describes their <strong>current circumstances</strong>"
+msgstr "Gin tha <strong>coronavirus</strong>smittal haes made odds til thair oors o wark, ootwale tha repone 'at gies a richt insensin o thair<strong> stannin thenoo</strong>"
```
Comment (berroar) [inline translations/eo/census_household_gb_nir_eo.po:1754]:
```suggestion
msgstr "Gin tha <strong>coronavirus</strong> smittal haes made odds til thair oors o wark, ootwale tha repone 'at gies a richt insensin o thair<strong> stannin thenoo</strong>"
```

Comment (petechd) [reply translations/eo/census_household_gb_nir_eo.po]: Added, thanks!

File: translations/eo/census_household_gb_nir_eo.po [hunk 12]
```diff
@@ -2,4303 +2,6491 @@ msgid ""
...
+
+#. Question definition link
+msgctxt "What is <em>{person_name_possessive}</em> main language?"
+msgid "What we mean by “main language”"
+msgstr "Whut we mean by \"foremaist leid\""
+
+#. Question definition description
+msgctxt "Do you usually live at {household_address}?"
+msgid "This is often your permanent or family home."
+msgstr "This bes affen yer bidin or faimlie hame"
```
Comment (MebinAbraham) [inline translations/eo/census_household_gb_nir_eo.po:1864]:
```suggestion
msgstr "This bes affen yer bidin or faimlie hame."
```

**Review by MebinAbraham — APPROVED** [review]

**Review by berroar — APPROVED** [review]

**Pull request merged**